package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cowprotocol/cow-sdk",
-  "version": "6.0.0-RC.36",
+  "version": "v5.11.0-RC.0",
```

Comment:
Is it intended?

Reply:
Yes -> https://github.com/cowprotocol/cow-sdk/pull/303#discussion_r2086573872

Do you think we should keep the v6?

Also, here it shouldn't have the `v` prefix.

Reply:
Aaah, now I understand.
Since the latest (not RC) release is 5, it makes sense to keep 5.
Thanks for pointing.